HVCAEN/iocBoot/iocHVCAENSIM-IOC-01/st.cmd

```diff
@@ -27,7 +29,8 @@ CAENx527ConfigureCreate "hv0", "127.0.0.1"
 
 ## Load our record instances
 #dbLoadRecords("db/xxx.db","user=faa59Host")
-CAENx527DbLoadRecords("P=$(MYPVPREFIX)CAEN, ALARM_WHEN_ON=$(ALARM_WHEN_ON=NO_ALARM), ALARM_WHEN_RAMPING=$(ALARM_WHEN_RAMPING=NO_ALARM)")
+$(CAN_WRITE= ) CAENx527DbLoadRecords("P=$(MYPVPREFIX)CAEN, ALARM_WHEN_ON=$(ALARM_WHEN_ON=NO_ALARM), ALARM_WHEN_RAMPING=$(ALARM_WHEN_RAMPING=NO_ALARM), ASG=READONLY")
+$(IFREADONLY= ) CAENx527DbLoadRecords("P=$(MYPVPREFIX)CAEN, ALARM_WHEN_ON=$(ALARM_WHEN_ON=NO_ALARM), ALARM_WHEN_RAMPING=$(ALARM_WHEN_RAMPING=NO_ALARM), ASG=DEFAULT")
```

Comment:
Using this approach we have to duplicate lines in a number of places, this makes the code harder to change in future. How about just passing down the ASG directly from the config (restricted to just `READONLY` and `DEFAULT` by the pattern)? This way we don't need to worry about any extra code. I also thing `READONLY`/`DEFAULT` are clearer options for a user than `#`/` ` (even with an explanation).